themes/aura/assets/product.js

```diff
@@ -587,7 +609,7 @@ function setup() {
     );
 
     if (productDetails) {
-      const observer = new MutationObserver(() => {
+      const observer = new MutationObserver(async () => {
```

Comment:
This will be called anytime the user clicks _anywhere_ inside the element the observer is observing.

Reply:
https://github.com/user-attachments/assets/a10e5905-0798-4978-9511-b772b9c69f98

Reply:
Sorry but I disagree with you. This observer will be called anytime the user clicks on a variant not anywhere inside the element the observer is observing.

Reply:
Tested it in test and princess and seems to be working as expected; The request is sent when a form control is clicked. Can you check if the same thing happens in prod?

Reply:
Yes I have tested it in production and it's working fine.

Reply:
Can you add the same variant options as this product and test again please: https://eihab.youcan.store/products/iphone-15-pro-max

Reply:
after deep testing yes I found out that the dropdown variant that was making this issue. thanks fix pushed